README.md

```diff
@@ -28,11 +28,17 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
   - [Want to hack on IPFS?](#want-to-hack-on-ipfs)
 - [License](#license)
 
-## Use Cases
+## Platforms
```

Comment:
Well, dapple, digx and ujo I haven't seen before, and their websites are not overly clear on what it is either, so I'm unsure. I wouldn't be opposed of scraping the `Platforms` category in favor of just listing them as apps right now.